src/overlays/actors/ovl_Door_Ana/z_door_ana.c

```diff
@@ -122,7 +122,7 @@ void DoorAna_WaitClosed(DoorAna* this, PlayState* play) {
         play_sound(NA_SE_SY_CORRECT_CHIME);
     }
 
-    func_800B8C50(&this->actor, play);
+    Actor_SetClosestSecretDistance(&this->actor, play);
```

Comment:
???

Reply:
What's wrong?

Reply:
Sorry I misread last night, this comment can be ignore

Tangent: Does `closestSecretDistSq` not get used by anything? A quick search shows nothing.

Reply:
It is used by player :slightly_smiling_face: 
MM: https://github.com/Decompollaborate/mm/blob/1609388cec22a7110284d53f0cefc71603c6221c/src/overlays/actors/ovl_player_actor/z_player.c#L11224
OoT: https://github.com/zeldaret/oot/blob/83c3b64b1c267f13f2839a6ec18a4947ca70b0ef/src/overlays/actors/ovl_player_actor/z_player.c#L10429
It is basically the same function in both games, the only difference is OoT checks for Stone of Agony